livre/eq/Config.cpp

```diff
@@ -224,24 +220,9 @@ bool Config::frame()
     // reset data and advance current frame
     frameSettings->setGrabFrame( false );
 
-    if( params.animation != INT_MAX )
-    {
-        const uint32_t start = frameMin;
-        const uint32_t end = std::max( frameMax, current );
-        const int32_t delta = params.animation;
-
-        // avoid overflow condition:
-        const uint32_t fullInterval = FULL_FRAME_RANGE[ 1 ] - FULL_FRAME_RANGE[ 0 ];
-        const uint32_t interval = end - start == fullInterval ?
-                                      fullInterval : end - start + 1;
-        // If current is at the beginning and animation is reverse,
-        // set current to the end
-        if(( current - start == 0 ) && ( delta < 0 ))
-            current = end;
-
-        const uint32_t frameNumber = (( current - start + delta ) % interval ) + start;
-        frameSettings->setFrameNumber( frameNumber );
-    }
+    if( !keepToLatest )
+        frameSettings->setFrameNumber(
```

Comment:
This place is still a mess  ( not yours, my or others fault ) :) Why do we get the next frame and current frame does not make sense at all. The frame util class could always provide the current frame knowing the previous one. But lets keep it like this now.